### DISCUSSION TITLE
Generate notes between two builds automatically

### DISCUSSION THREAD

**Discussion by Alexandre-NAUD** [rfennell/AzurePipelines#1557] [Q&A] [answered]
Hi,

We use scrum and 4 weeks sprint for our project. At the end of a sprint, we made a release branch tagged with version like '1.0.1', '1.0.2'... and we would like to generate release notes between last release branch and sprint n-1 release branch.

**Comment by rfennell** [comment, marked as answer]
I fear that might not be possible, I can't remember of the top of my head if the AZDO API allows you to get the differences between two pipeline runs on different branches

To test it I would do the following

- Set `checkstages=true`
- Set the `overrideBuildReleaseId=1234` where the number of the pipeline run ID for the `1.0.1` run

You would need to update the `overrideBuildReleaseId` variable each sprint, but that should be the only manual operations

Let me know how it goes

**Comment by Alexandre-NAUD** [comment]
Hi,

It works correctly between 2 releases but not between 2 builds. If I use buildID with checkstages=true, and 2 builds with just some works items, it take severals minutes to get results and return hundreds of WI.

**Reply by rfennell** [reply]
Yes, volume of associated items can be a problem, hope you find what returns useful
